manifests/s/Stoplight/Studio/2.8.1/Stoplight.Studio.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Stoplight.Studio
+PackageVersion: 2.8.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/stoplightio/studio/releases/download/v2.8.1-stable.7959.git-777e356/stoplight-studio-win.exe
+  InstallerSha256: 626E61EE6FCAB3F39A3FC477D327FF91C6943FB020F9F7E47105FFD61ADD1514

```

```suggestion
  InstallerSha256: 626E61EE6FCAB3F39A3FC477D327FF91C6943FB020F9F7E47105FFD61ADD1514
  ProductCode: 279242fd-d458-5566-9571-36711297e305
  AppsAndFeaturesEntries: 
  - DisplayName: Stoplight Studio 2.8.1-stable.7959.git-777e356
    DisplayVersion: 2.8.1-stable.7959.git-777e356
    ProductCode: 279242fd-d458-5566-9571-36711297e305
```
AppsAndFeaturesEntries are needed because the DisplayVersion in control panel does not match the Package Version

---

Got it. Thanks for the clarification @Trenly.